docs/rfcs/0002/readme.md

```diff
@@ -37,21 +34,98 @@ The proposal below might take a couple of read-throughs; I've also added a concr
-        2. duplicate the pod template specification from the Container Profile and patches the necessary Volumes and the config map. The spec is used as the basis for starting the necessary pods and containers.
-    3. The containerized application executes. The expected mounts are available per the requirements and celebration occurs.
-
-Example
--------
-
-Say I authored a simple application, `foo`, that requires Rabbit local GFS2 storage and a persistent Lustre storage volume. As the author, my program is coded to expect the GFS2 volume is mounted at `/foo/local` and the Lustre volume is mounted at `/foo/persistent`. In this case, the storages are not optional, so they are defined as such in the NNF Container Profile.
+    3. the NNF storage requirements described above.
+3. The ADMINISTRATOR creates a corresponding _NNF Container Profile_ Kubernetes custom resource with the necessary NNF storage requirements and pod specification as described by the AUTHOR
+4. The USER who desires to use the application works with the AUTHOR and the related NNF Container Profile to understand the storage requirements
+5. The USER submits a WLM job with the #DW container directive variables populated
+6. WLM runs the workflow and drives it through the following stages...
+    1. `Proposal`: RABBIT validates the #DW container directive by comparing the supplied values to those listed in the NNF Container Profile. If the workflow fails to meet the requirements, the job fails
+    2. `PreRun`: RABBIT software:
+        1. creates a config map reflecting the storage requirements and any runtime parameters; this is provided to the container at the volume mount named `nnf-config`, if specified
+        2. duplicates the pod template specification from the Container Profile and patches the necessary Volumes and the config map. The spec is used as the basis for starting the necessary pods and containers
+    3. The containerized application(s) executes. The expected mounts are available per the requirements and celebration occurs. The pods continue to run until:
+       1. a pod completes successfully (any failed pods will be retried)
+       2. the max number of pod retries is hit (indicating failure on all retry attempts)
+          1. Note: retry limit is non-optional per Kubernetes configuration
+          2. If retries are not desired, this number could be set to 0 to disable any retry attempts
+    4. `PostRun`: RABBIT software:
+       1. marks the stage as `Ready` if the pods have all completed successfully. This includes a successful retry after preceding failures
+       2. starts a timer for any running pods. Once the timeout is hit, the pods will be killed and the workflow will indicate failure
+       3. leaves all pods around for log inspection
+
+### Container Assignment to Rabbit Nodes
+
+During `Proposal`, the USER must assign compute nodes for the container workflow. The assigned
```

Comment:
Are you inserting these line breaks or am I looking at some GitHub rendering? Shouldn't need line breaks in a markdown file

Reply:
I'm breaking at 100 characters to make it easier to view/edit in text editor. It's makes it easier for me to use vim keybindings to edit files when lines are at reasonable length. Doesn't affect the rich formatting of the markdown though.

Reply:
Would the character limit wreak havoc with any number/listed items? And now you've introduced a different formatting pattern that requires others to have that same pattern

Reply:
I haven't touched any of the lists for that exact reason. I can undo this if it's a pain point - I realize I'm introducing inconsistency here.

Reply:
I removed them